packages/block-editor/src/components/font-appearance-control/index.js

```diff
@@ -144,11 +144,12 @@ export default function FontAppearanceControl( props ) {
 	}, [ props.options ] );
 
 	// Find current selection by comparing font style & weight against options.
-	const currentSelection = selectOptions.find(
-		( option ) =>
-			option.style.fontStyle === fontStyle &&
-			option.style.fontWeight === fontWeight
-	);
+	const currentSelection =
+		selectOptions.find(
+			( option ) =>
+				option.style.fontStyle === fontStyle &&
+				option.style.fontWeight === fontWeight
+		) || selectOptions[ 0 ];
```

Comment:
Thanks for testing @aaronrobertshaw, I've reverted this line and moved the fix over to the CustomSelectControl component. From taking a look at the error from downshift, it looks like setting the `selectedItem` to an empty string should do the trick ([here's](https://github.com/downshift-js/downshift/issues/478#issuecomment-455668762) where other folks have encountered the issue previously).

I've added in an optional chaining to another line that expected a currently selected item just in case, too.

Reply:
Thanks for the quick turnaround. In further testing of the old approach, I noticed that the initial change from no selection to a non-default option did not actually take effect.

After the latest updates to this PR, I'm seeing what appears to be a related issue but slightly different as now the default option displays as blank.

| Before | After |
|---|---|
| ![FontAppearanceIssue](https://user-images.githubusercontent.com/60436221/131957270-c4dca8f3-d193-47d0-bd3a-76529665a63b.gif) | ![FontAppearanceIssue5](https://user-images.githubusercontent.com/60436221/131957279-0021a093-8d9c-4613-be32-e71e1faa0e28.gif) |

Reply:
Good catch @aaronrobertshaw! Since it's near the end of our week, I've reverted the change to the `selectedItem` so that this PR becomes just a fix for the two areas where describedBy is expecting an object. I'll write up the downshift issue separately so I can keep digging into that next week.

Reply:
Added a follow-up issue linking back here: https://github.com/WordPress/gutenberg/issues/34524